api-report/debugger.api.md

```diff
@@ -128,7 +128,7 @@ export class DebugReplayController extends ReplayController implements IDebugger
 export namespace FluidDebugger {
     export function createFromService(documentService: IDocumentService): Promise<IDocumentService>;
     // (undocumented)
-    export function createFromServiceFactory(documentServiceFactory: IDocumentServiceFactory): Promise<IDocumentServiceFactory | ReplayDocumentServiceFactory>;
+    export function createFromServiceFactory(documentServiceFactory: IDocumentServiceFactory): Promise<ReplayDocumentServiceFactory | IDocumentServiceFactory>;
```

Comment:
is this expected to change?

Reply:
Two seem to be equivalent.

Reply:
@ssimic2 This is possibly the root of the build failure. I'm betting this function is defined without an explicit return type in the source, letting TypeScript infer the return type. TS and in turn api-extractor makes no guarantees about the order in which union types will be emitted, since either is correct from the TS perspective. We have an eslint rule in the strict config that requires all functions have explicit return types to help prevent this, but few packages use it.

The solution is to add a return type to the source code, which TS/api-extractor will then use.